scripts/build_cop_profiles/run.py

```diff
@@ -68,6 +147,23 @@ def get_cop(
 
     set_scenario_config(snakemake)
 
+    # map forward and return temperatures specified on country-level to onshore regions
+    regions_onshore = gpd.read_file(snakemake.input.regions_onshore)["name"]
```

Comment:
it is normally better to keep the snakemake object outside the function, so that one can call the function also from a different script

Reply:
Hm, not sure I can follow here. Isn't that what I'm doing? (`regions_onshore` is read from the snakmake input in the `__main__` block)